### PR TITLE
TELCODOCS-903-RN: Release note for Graceful Node Shutdown 

### PR DESCRIPTION
**Pull request by tmulquee**
Version(s): 4.13 only

Issue: https://issues.redhat.com/browse/TELCODOCS-903

Link to docs preview: https://55869--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-nodes

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -86,6 +86,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-nodes"]
 === Nodes
 
+==== Graceful Node Shutdown
+
+Graceful Node Shutdown (GNS) enables kubelet to delay forcible eviction of pods in the event of a node shutdown.
+This is important for users who want to ensure that workloads that execute critical operations cannot be interrupted.
```
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:92]:
```suggestion
This ensures that workloads that execute critical operations are not interrupted.
```

**Review by aireilly — COMMENTED** [review]
Added a few small comments for clarity and readability.

The complete suggested rewrite is as follows: 

```asciidoc
[id="ocp-4-13-graceful-node-shutdown"]
==== Graceful node shutdown
Graceful node shutdown enables kubelet to delay forcible eviction of pods in the event of a node shutdown.
This ensures that workloads that execute critical operations are not interrupted.

You can specify a termination grace period for protected workloads.
This ensures that the pod shuts down in an orderly manner, including allowing operations that are monitoring the activity of sensitive workloads to finish.
For example, if an operator that flashes network card firmware is shut down in a disorderly fashion, hardware can be damaged.
You can also tag workloads to specify the order of shut down.
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -86,6 +86,18 @@ This release adds improvements related to the following components and concepts.
...
+For example, if an operator that flashes network card firmware is shut in a disorderly fashion, hardware can be damaged.
```
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:93]:
```suggestion
For example, if an operator that flashes network card firmware is shut down in a disorderly fashion, hardware can be damaged.
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -86,6 +86,18 @@ This release adds improvements related to the following components and concepts.
...
+
+GNS gives admins the ability to specify a termination grace period for protected workloads.
+This ensures time for the pod to shut down in an orderly manner, including operations that are monitoring the activity of sensitive workloads.
```
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:96]:
```suggestion
This ensures that the pod shuts down in an orderly manner, including allowing operations that are monitoring the activity of sensitive workloads to finish.
```

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:93]: Suggested rewrite for clarity: Delete this example and add it below alongside the description of the function.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -86,6 +86,18 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-nodes"]
 === Nodes
 
+==== Graceful Node Shutdown
```
Comment (aireilly) [inline release_notes/ocp-4-13-release-notes.adoc:89]:
```suggestion
[id="ocp-4-13-graceful-node-shutdown"]
==== Graceful Node Shutdown
```

**Comment by kalexand-rh** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/58197.

**Pull request closed without merging**